ext/standard/html.c

```diff
@@ -1559,3 +1560,125 @@ PHP_FUNCTION(get_html_translation_table)
 	}
 }
 /* }}} */
+
+static zend_result cache_request_parse_body_option(HashTable *options, zval *option, int cache_offset)
+{
+	if (option) {
+		zend_long result;
+		if (Z_TYPE_P(option) == IS_STRING) {
+			zend_string *errstr;
+			result = zend_ini_parse_quantity(Z_STR_P(option), &errstr);
+			if (errstr) {
+				zend_error(E_WARNING, "%s", ZSTR_VAL(errstr));
```

Comment:
My 2 cents: since this is new code, and setting invalid values for ini configs is definitely a programmatic error, I'd vote for using exceptions here instead of warnings. Or do you have any argument against this idea which I'm not aware of?

Reply:
Makes sense :+1:

Reply:
I actually opted against this anyway, because the API of `zend_ini_parse_quantity` doesn't offer a way to disable the fallback behavior. The error message would be out of place, throwing with "falling back to x for compatibility". So let's keep this behavior consistent, and then making it throw across the board.